tomcatContainerImage/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rockylinux:9
+# Make it uptodate
+RUN dnf -y update
+# Install wget for downloading tomcat package
+RUN dnf install wget -y && dnf clean all
+# Rocky linux 9 does not come with java at all. Thats why java-11 install
+RUN dnf install java-11-openjdk-headless -y && dnf clean all
```

Comment:
I think these could be condensed into a single run statement,
`dnf update -y && dnf install wget java-11-openjdk-headless -y && dnf clean all`